compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/codegen/ExpressionCodegen.kt

```diff
@@ -202,6 +208,21 @@ class ExpressionCodegen(
         )
     }
 
+    private fun writeReceiverParameterInLocalVariableTable(param: IrValueParameter, startLabel: Label, endLabel: Label) {
+
+        //TODO: get rid of descriptor
+        val name = getNameForReceiverParameter(
+            irFunction.descriptor,
+            typeMapper.kotlinTypeMapper.bindingContext,
+            context.configuration.languageVersionSettings)
+
+        val type = typeMapper.mapType(param)
+        typeMapper.kotlinTypeMapper.bindingContext
```

Comment:
Looks like this line has no side-effects. Was it added on purpose?

Reply:
> Looks like this line has no side-effects. Was it added on purpose?

Should be a mistake I reckon, I'll remove it.